plugins/aws-s3-storage/package.json

```diff
@@ -36,6 +36,7 @@
     "@verdaccio/babel-preset": "^9.0.0",
     "@verdaccio/eslint-config": "^9.0.0",
     "@verdaccio/types": "^9.0.0",
+    "jest": "^24.9.0",
```

Comment:
We have `jest` already in the root. do we need this here for some reason?

Reply:
Furthermore, worth to update `aws-sdk@latest: 2.607.0` , does make sense to you?

Reply:
regarding `jest`, it is a mixed bag and I have my own difficulties in our own projects since we are also using lerna. Sometimes as a developer I don't open up the root project in my IDE, rather just the child project. And my IDE(intellij) does not give me a test runner until I add `jest` as a dependency. But ultimately it's up to you guys since its more of a style guide issue.

Reply:
regarding `aws-sdk` I agree we can update